src/main/java/net/dv8tion/jda/internal/entities/ReceivedMessage.java

```diff
@@ -332,6 +334,15 @@ public Member getMember()
         return member;
     }
 
+    @Override
+    public int getApproximatePosition()
+    {
+        if (!getChannelType().isThread())
+            throw new IllegalStateException("This message was not sent in a thread.");
```

Comment:
```suggestion
        if (!getChannelType().isThread()) 
        {
            throw new IllegalStateException("This message was not sent in a thread.");
        }
```
I would place curly brackets here to remain consistent with the JDA [code style](https://github.com/DV8FromTheWorld/JDA/wiki/6%29-JDA-Structure-Guide#indentation-and-brackets)

Reply:
Maintainers does not want pointless spaces, such as these one liners